scRNA-seq-advanced/04-differential_expression.Rmd

```diff
@@ -12,156 +12,260 @@ output:
+Now when we look at the `colData` of the SCE object we should see new columns, including the `diagnosis_group` column which indicates if each cell comes from an ERMS or ARMS sample.
+We can now use that column to label any UMAP plots (or other plot types) that we make.
+In the chunk below we will start by taking a look at our integration results and color our cells by RMS subtype.
 
-```{r}
+**Reminder: You should always use the batch-corrected dimensionality reduction results for visualizing datasets containing multiple libraries or samples.**
+
+```{r diagnosis group UMAP}
+# UMAP of all samples, separating by diagnosis group
 scater::plotReducedDim(integrated_sce,
                        dimred = "fastmnn_UMAP",
-                       colour_by = "sample",
+                       colour_by = "diagnosis_group",
                        point_size= 0.5,
-                       point_alpha = 0.2,
-                       other_fields = "diagnosis_group") +
-  facet_wrap(~diagnosis_group)
+                       point_alpha = 0.2) 
 ```
 
-Interestingly, it looks like samples from the ARMS and ERMS subdiagnoses tend to integrate with each other rather than integrate all together. 
+Interestingly, it looks like samples from the ARMS and ERMS subtypes tend to cluster with samples of the same subtype rather than cluster all together. 
 
-We can also look at the distribution of cell types to see if these subdiagnoses have different or shared cell types.
+In the integration notebook we also looked at the distribution of cell types after integration.
+In that notebook we discussed that cells of the same cell type are expected to integrate with other cells of the same type.
+Is that the case with this dataset?
 
-*Talk about looking across cell types and comparing cell types* 
+A word of caution when evaluating the cell type results for this dataset: The cell types for this dataset were assigned in a two stage process as described in [Patel _et al._ (2022)](https://doi.org/10.1016/j.devcel.2022.04.003).
+The first stage assigned cells as tumor or non-tumor.
+The next stage further classified tumor cells into type of tumor cell (i.e., myoblast, myocyte, or mesoderm).
+Some samples could not be further classified, so all of their tumor cells are denoted `Tumor`.
+The samples which could be further classified have a mix of `Tumor_Mesoderm`, `Tumor_Myoblast`, and `Tumor_Myocyte`.
 
-```{r}
-# facet the UMAP by sample to show the distribution of cell types across each subdiagnosis
+```{r celltype UMAP}
+# # UMAP of all libraries, separating by diagnosis group and labeling cell type
 scater::plotReducedDim(integrated_sce,
                        dimred = "fastmnn_UMAP",
+                       # color each point by cell type
+                       colour_by = "celltype_broad",
+                       point_size= 0.5, 
+                       point_alpha = 0.4)
+```
+
+That doesn't appear to be the case with this dataset. 
```

Comment:
Since we detoured to talk about tumor cell types I had forgotten by now what "that" meant! Can we just re-contextualize this sentence a bit?